extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -49,17 +49,6 @@ const SUPPORTED_CSS_GRID_ATTRIBUTES_SELECTOR = Object.keys(
  */
 const TEMPLATE_ATTRIBUTE_NAME = 'template';
 
-/**
- * A mapping of template attribute values to CSS class names.
- * @const {!Object<string, string>}
- */
-export const GRID_LAYER_TEMPLATE_CLASS_NAMES = {
-  'fill': 'i-amphtml-story-grid-template-fill',
-  'vertical': 'i-amphtml-story-grid-template-vertical',
-  'horizontal': 'i-amphtml-story-grid-template-horizontal',
-  'thirds': 'i-amphtml-story-grid-template-thirds',
-};
```

Comment:
This is used by in animation-presets.js. it's `setStyleForPreset` method might need to be updated to read attributes instead of class names.

Reply:
True, will update that. Thanks!

Reply:
Looking at the `setStyleForPreset`, it doesn't seem like those checks are needed there, they are essentially removing the `...-fill` class from the template if the parent of the `amp-img` is `template="fill"` but this can be done with a CSS selector that's `[template="fill"]:not(.class-we-are-using-anyways)` without involving JS. Will commit these changes once I'm sure.